### DISCUSSION TITLE
save included and excluded files inputs state in globals

### DISCUSSION THREAD

**Discussion by 4c656f** [zed-industries/zed#7981] [Show and tell]
I often use the project search and faced the problem that for each new search it is necessary to reenter the included files path  every time, would it be better if the state of the inputs was saved in globals and the value was restored when the new search was opened?
current: 

https://github.com/zed-industries/zed/assets/85838594/a0dd157e-3595-44a9-bceb-0ebdf056bc8b

proposal:

https://github.com/zed-industries/zed/assets/85838594/6c49fc94-f2dd-4766-a461-7a80747fd034

**Comment by MishaKazakov** [comment]
Yes that's really necessary for my projects. And preserve search filters based on folder like VS Code
